beacon-chain/sync/initial-sync/sync_state.go

```diff
@@ -96,11 +97,11 @@ func (s *InitialSync) processState(msg p2p.Message) {
 }
 
 // requestStateFromPeer requests for the canonical state, finalized state, and justified state from a peer.
-func (s *InitialSync) requestStateFromPeer(ctx context.Context, lastFinalizedRoot [32]byte) error {
+func (s *InitialSync) requestStateFromPeer(ctx context.Context, lastFinalizedRoot [32]byte, peerID peer.ID) error {
```

Comment:
you should just use s.bestPeer here, since we are only going to sync with one peer